soak-test/src/common/config.ts

```diff
@@ -0,0 +1,71 @@
+import { NonEmptyString } from "@pagopa/ts-commons/lib/strings";
+import { IntegerFromString, NumberFromString } from "@pagopa/ts-commons/lib/numbers";
+import { BooleanFromString } from "@pagopa/ts-commons/lib/booleans";
+import { readableReport } from "@pagopa/ts-commons/lib/reporters";
+import * as E from "fp-ts/lib/Either";
+import * as t from "io-ts";
+import { pipe } from "fp-ts/lib/function";
+
+export const K6Config = t.interface({
+  rate: IntegerFromString,
+  duration: NonEmptyString,
+  rampingDuration: NonEmptyString,
+  preAllocatedVUs: IntegerFromString,
+  maxVUs: IntegerFromString,
+});
+export type K6Config = t.TypeOf<typeof K6Config>;
+
+export type IConfig = t.TypeOf<typeof IConfig>;
+export const IConfig = t.intersection([
+  t.interface({
+    URL_BASE_PATH: NonEmptyString,
+    API_SUBSCRIPTION_KEY: NonEmptyString,
+    API_SUBSCRIPTION_KEY_GENERATE_CONTRACT: NonEmptyString,
+    URL_BASE_PATH_GENERATE_CONTRACT: NonEmptyString,
+    DELETE_RATIO: NumberFromString,
```

Comment:
Maybe add a check that this is between 0 and 1?

Reply:
Solved by https://github.com/pagopa/pagopa-wallet-tests/pull/33/commits/da855ea0c5f593a13180c5c0544d779fe4729246